.gitignore

```diff
@@ -0,0 +1,5 @@
+# Default ignored files
```

Comment:
+1 for .gitignore but lost *.iml pattern

Reply:
Done